README.md

```diff
@@ -2,17 +2,63 @@
 
 [![Circle CI](https://circleci.com/gh/rit-sse/node-api.svg?style=svg&circle-token=50819f36da32c91bfd2df83ccae75175c0ff9a6e)](https://circleci.com/gh/rit-sse/node-api)
 
-The SSE new and improved unified API
-
-[Apiary Docs](http://docs.sse.apiary.io)
+The SSE new and improved unified API. Used in conjunction with the [OneRepoToRuleThemAll](https://github.com/rit-sse/OneRepoToRuleThemAll) project.
```

Comment:
I'd make a note about api-client and how you can _technically_ use it for other projects.